google/api_core/extended_operation.py

```diff
@@ -114,7 +122,7 @@ def error_message(self):
     def __getattr__(self, name):
         return getattr(self._extended_operation, name)
 
-    def done(self, retry=polling.DEFAULT_RETRY):
+    def done(self, retry=None):
         self._refresh_and_update(retry)
```

Comment:
What happens if user wants to use a different polling other than DEFAULT_RETRY ?